src/api/server/sync_io.rs

```diff
@@ -625,39 +625,31 @@ impl<F: FileSystem + Sync, D: AsyncDrive> Server<F, D> {
             return ctx.reply_ok(Some(out), None);
         }
 
-        // These fuse features are supported by this server by default.
-        let supported = FsOptions::ASYNC_READ
-            | FsOptions::PARALLEL_DIROPS
-            | FsOptions::BIG_WRITES
-            | FsOptions::AUTO_INVAL_DATA
-            | FsOptions::ASYNC_DIO
-            | FsOptions::HAS_IOCTL_DIR
-            | FsOptions::MAX_PAGES
-            | FsOptions::EXPLICIT_INVAL_DATA
-            | FsOptions::PERFILE_DAX;
-
         let capable = FsOptions::from_bits_truncate(flags);
 
         match self.fs.init(capable) {
             Ok(want) => {
-                let enabled = capable & (want | supported);
+                let enabled = capable & want;
```

Comment:
is there any impact for PassthroughFs? I saw it has implementation for Filesystem and if supported removed here the result enabled flags may be different

Reply:
It still works as long as it is attached via vfs. It is broken if it is directly attached but that is not how we are using passthroughFS right now.

Reply:
ok, may need some notice on release note for that change in next version